index.js

```diff
@@ -1,23 +1,39 @@
-const { exec } = require('child_process');
+const {exec} = require('child_process');
 
 let curPid;
 let uids = {};
+// for windows support
+let curDir;
+const dirRegex = /([\w]:[\s\S]+)>/;
 
-const setCwd = (pid) =>
-  exec(`lsof -p ${pid} | grep cwd | tr -s ' ' | cut -d ' ' -f9-`, (err, cwd) => {
-    if (err) {
-      console.error(err);
-    } else {
-      cwd = cwd.trim();
+const setCwd = (pid) => {
+  if (process.platform === 'win32') {
+    if (curDir) {
       store.dispatch({
         type: 'SESSION_SET_CWD',
-        cwd
+        cwd: curDir
       });
     }
-  });
+  } else {
+    exec(`lsof -p ${pid} | grep cwd | tr -s ' ' | cut -d ' ' -f9-`, (err, cwd) => {
+      if (err) {
+        console.error(err);
+      } else {
+        cwd = cwd.trim();
+        store.dispatch({
+          type: 'SESSION_SET_CWD',
+          cwd
+        });
+      }
+    });
+  }
+}
 
 exports.middleware = (store) => (next) => (action) => {
   switch (action.type) {
+    case 'SESSION_ADD_DATA':
+      if (process.platform === 'win32' && dirRegex.test(action.data)) curDir = dirRegex.exec(action.data)[1];
```

Comment:
seems like you could call setCwd with the current directory here (`dirRegex.exec(action.data)[1]`).